pages/about.html

```diff
@@ -0,0 +1,307 @@
+    </div>
+    <div class="middle-part">
+      <menu class="mobile-list">
+        <li>
+          <a href="../index.html">
+            Home
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            About
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            Join
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            Sponsor
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            News
+          </a>
+        </li>
+      </menu>
+      <button type="button" class="campaign-button button">
+        WHO Campaign
+      </button>
+    </div>
+    <ul class="account-bar">
+      <li>
+        <a href="#">
+          <i class="fab fa-facebook-f"></i>
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <i class="fab fa-twitter"></i>
+        </a>
+      </li>
+      <li>
+        <a class="third-child" href="#">
+          English
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          My Page
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          Logout
+        </a>
+      </li>
+    </ul>
+  </div>
+
+  <!-- absolutely positioned elements -->
+
+  <header>
+
+    <div class="mobile-nav">
+      <button class="mobile-menu-button" type="button">
+        <i class=" fas fa-bars fa-2x"></i>
+      </button>
+    </div>
+    <div class="desktop-nav">
+      <ul class="account-bar">
+        <li>
+          <a href="#">
+            <i class="fab fa-facebook-f"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <i class="fab fa-twitter"></i>
+          </a>
+        </li>
+        <li>
+          <a class="third-child" href="#">
+            English
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            My Page
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            Logout
+          </a>
+        </li>
+      </ul>
+      <nav class="nav-bar">
+        <a class="logo" href="../index.html">
+          <img src="../images/h-logo-white.svg" alt="logo">
+           
+        </a>
+        <div class="page-navigation">
+          <ul class="pages">
+            <li>
+              <a href="../index.html">
+                Home
+              </a>
+            </li>
+            <li>
+              <a class="current-page" href="#">
+                About
+              </a>
+            </li>
+
+            <li>
+              <a href="#">
+                Join
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Sponsor
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                News
+              </a>
+            </li>
+          </ul>
+          <button type="button"
+            class="compaign-button button">
+            WHO Campaign
+          </button>
+        </div>
+      </nav>
```

Comment:
- [ ] The blue color from the World Health Organization is very different from the color theme from the template you used. A white background for the navbar is more appropriate. Kindly change that.
Template:
<img width="741" alt="image" src="https://user-images.githubusercontent.com/61919617/181599726-16fd2e45-265f-4d67-8fc8-917e2edf838b.png">